sagemaker-spark-sdk/src/main/scala/com/amazonaws/services/sagemaker/sparksdk/transformation/deserializers/ProtobufResponseRowDeserializer.scala

```diff
@@ -166,3 +166,29 @@ class LinearLearnerRegressorProtobufResponseRowDeserializer
     StructType(Array(StructField(scoreColumnName, DoubleType))),
     protobufKeys = Some(Seq("score")))
 
+/**
+  * Deserializes a Protobuf response from the FactorizationMachines model image with predictorType
+  *   "binary_classifier" into Rows in a Spark Dataframe
+  * @param scoreColumnName name of the column of Doubles indicating the output score for the record.
+  * @param predictedLabelColumnName name of the column of Doubles indicating the predicted label
+  *   for the record.
+  */
+class FactorizationMachinesBinaryClassifierProtobufResponseRowDeserializer
```

Comment:
We have follow this naming schema for the others, but a class name this long still looks odd. What do you think about calling this something like `FMBinaryClassifierDeserializer`? It would be inconsistent with the others, but this minor naming inconsistency wouldn't offend me as much as this really long class name.

(In hindsight, we really should have shrunk the names of these before, like "ResponseRowDeserializer" name to just "Deserializer" and dropped the "Protobuf" qualifier, too.)

Reply:
I didn't like name that long too. I wouldn't shorten to FM, so maybe:
FactorizationMachinesBinaryClassifierDeserializer and
FactorizationMachinesRegressorDeserializer

isn't too long after all?

Reply:
Yeah, those are reasonable.